content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -17,6 +17,22 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+## Specify my customerize folder
+
+Set the env `GITEA_CUSTOM`.
```

Comment:
Specify that not setting it defaults to a `custom` directory and where such directory is resolved when relative (like in the default case)